docs/source/cluster_management/clis.rst

```diff
@@ -37,7 +37,10 @@ The following useful commands come with the vanilla atlasdb-cli.
 sweep
 -----
 
-Sweep old table rows. This can be useful for improving performance if having too many dead cells is impacting read times.  The command allows you to specify a namespace or a specific set of tables. Run ``./bin/atlasdb help sweep`` for more information.
+Sweep old table rows.
+This can be useful for improving performance if having too many dead cells is impacting read times.
+The command allows you to specify a namespace or a specific set of tables.
+Run ``./bin/atlasdb help sweep`` for more information, or check out :ref:`the sweep CLI documentation <atlas_sweep_cli>`.
```

Comment:
Consider reordering: "For more information, see the sweep CLI documentation or run ./bin/atlasdb help sweep."